cognite/client/_api/relationships.py

```diff
@@ -0,0 +1,172 @@
+                to return all items.
+
+        Returns:
+            RelationshipList: List of requested relationships
+
+        Examples:
+
+            List relationships::
+
+                >>> from cognite.client import CogniteClient
+                >>> c = CogniteClient()
+                >>> relationship_list = c.relationships.list(limit=5)
+
+            Iterate over relationships::
+
+                >>> from cognite.client import CogniteClient
+                >>> c = CogniteClient()
+                >>> for relationship in c.relationships:
+                ...     relationship # do something with the relationship
+
+            Iterate over chunks of relationships to reduce memory load::
+
+                >>> from cognite.client import CogniteClient
+                >>> c = CogniteClient()
+                >>> for relationship_list in c.relationships(chunk_size=2500):
+                ...     relationship_list # do something with the relationships
+        """
+        return self._list(method="GET", limit=limit, filter=None)
+
+    def create(self, relationship: Union[Relationship, List[Relationship]]) -> Union[Relationship, RelationshipList]:
+        """Create one or more relationships.
+
+        Args:
+            relationship (Union[Relationship, List[Relationship]]): Relationship or list of relationships to create.
+                Note: the source and target field in the Relationship(s) can be of the form shown below, or objects of type Asset, TimeSeries, FileMetadata, Event
+
+        Returns:
+            Union[Relationship, RelationshipList]: Created relationship(s)
+
+        Examples:
+
+            Create a new relationship::
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import Relationship
+                >>> c = CogniteClient()
+                >>> rel = Relationship(external_id="rel",source={"resource":"TimeSeries", "resourceId": "ts"},target={"resource":"Asset", "resourceId": "a"},relationship_type="belongsTo",confidence=0.9,data_set="ds_name")
+                >>> res = c.relationships.create(rel)
+
+            Create a new relationship::
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import Relationship
+                >>> c = CogniteClient()
+                >>> assets = c.assets.retrieve_multiple(id=[1,2,3])
+                >>> flowrel1 = Relationship(external_id="flow_1",source=assets[0],target=assets[1] ,relationship_type="flowsTo",confidence=0.1,data_set="ds_flow")
+                >>> flowrel2 = Relationship(external_id="flow_2",source=assets[1],target=assets[2] ,relationship_type="flowsTo",confidence=0.1,data_set="ds_flow")
+                >>> res = c.relationships.create([flowrel1,flowrel2])
+        """
+        utils._auxiliary.assert_type(relationship, "relationship", [Relationship, list])
```

Comment:
Should we let this be a `RelationshipList` aswell?

Reply:
I thought we generally didn't have a solution for this (as in - cleaning retrieved objects and copying them)